projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/agents-button/agents-button.component.spec.ts

```diff
@@ -291,10 +302,32 @@ describe('AgentsButtonComponent', () => {
         expect(getAvatar('2')).toBeTruthy();
       });
 
-      it('should assign correct initials to each avatar for each agent', () => {
+      it('should assign correct initials to each avatar for each agent with double section name', () => {
         expect(getAvatar('1').initials).toBe('HA');
         expect(getAvatar('2').initials).toBe('PA');
       });
+
+      it('should assign correct initials to each avatar for each agent with single section name', () => {
+        const agentsWithSingleSectionName = [
```

Comment:
There are already available agents in 'agentsWithAvatar' variable. You can reuse them by just modifying 'name' field probably. Something like: 
agentsWithAvatar[0].name = 'Adam';
agentsWithAvatar[1].name = 'Bob';

Thanks that we will need to update only 2 fields instead of creating whole array again.